graph/is_graph_bipartite.cpp

```diff
@@ -37,127 +37,124 @@
  * @namespace graph
  * @brief Graph algorithms
  */
-namespace graph{
-	/**
-	 * @namespace is_graph_bipartite
-	 * @brief Functions for checking whether a graph is bipartite or not
-	 */
-	namespace is_graph_bipartite{
-		/**
-		 * @brief Class for representing graph as an adjacency list.
-		 */
-		class Graph {
- 			private: 
- 				int n;						/// size of the graph
-
-	    		std::vector<std::vector <int> > adj;	/// adj stores the graph as an adjacency list
-
-		    	std::vector<int> side;			///stores the side of the vertex
+namespace graph {
+  /**
+   * @namespace is_graph_bipartite
+   * @brief Functions for checking whether a graph is bipartite or not
+   */
+  namespace is_graph_bipartite {
+    /**
+     * @brief Class for representing graph as an adjacency list.
+     */
+    class Graph {
+      private:
+        int n;            /// size of the graph
```

Comment:
```suggestion
        int n;            ///< size of the graph
```
the comment style for class members is [slightly different](https://www.doxygen.nl/manual/docblocks.html#memberdoc)